polygon/rest/models/aggs.py

```diff
@@ -4,24 +4,105 @@
 
 @dataclass
 class Agg:
+    open: float
+    high: float
+    low: float
+    close: float
+    volume: float
+    vwap: float
     timestamp: int
+    transactions: int
+
+    @staticmethod
+    def from_dict(d):
+        return Agg(
+            d.get("o", None),
```

Comment:
Zack showed me that you can make this concise by doing
@staticmethod def from_dict(d): return Agg(**d)
which works when the return attributes are longer than one character (like in this case). NBD, just wanted to say in case you prefer it that way. Github erased my newlines in the code, but you can see examples of this in the other files in the v1 branch.

Reply:
I don't think you can do it here when the name `"open"` doesn't match the dict object `"o"`.